web/cobrands/sass/_mixins.scss

```diff
@@ -10,20 +22,20 @@ $direction: 'left' !default;
     $nav_background_colour: $nav_fg_hover !global;
 }
 
-@mixin button-size() {
-    display: inline-block;
-    margin: 0;
-    padding: 0.75em 1em;
+
+@mixin button-size($button-margin-top:$button-margin-top, $button-margin-right:$button-margin-right, $button-margin-bottom:$button-margin-bottom, $button-margin-left:$button-margin-left, $button-padding-top:$button-padding-top, $button-padding-right:$button-padding-right, $button-padding-bottom:$button-padding-bottom, $button-padding-left:$button-padding-left) {
+    @include flex-container-inline();
```

Comment:
I think @zarino's comment about flex inline was for applying to the geolocate link svg bit - https://github.com/mysociety/fixmystreet/pull/3291/files#diff-3ae05a9e3e785d418edb2ae5a8abad31d99b8844cef4fbcee18e4a932779633bR2474 - not all button-sizes, unless you wish to go through and check this hasn't affected anything else (I think this is used by every button on the site :) ).

Reply:
Yeah, @mySocietyClive you’ve added `@include flex-align($alignment: center);` to the `button-size()` mixin, but I don’t think we want it there, just on the geolocation button, after you `@include flex-container-inline();`.

So, yeah, remove the `flex-align` line from the `button-size` mixin in `_mixins.scss`, and put it into the `a#geolocate_link` ruleset in `_base.scss` :+1: